app/services/schools/advice/consumption_by_month_service.rb

```diff
@@ -3,33 +3,49 @@
 module Schools
   module Advice
     module ConsumptionByMonthService
-      def self.consumption_by_month(aggregate_meter)
-        end_date = aggregate_meter&.amr_data&.end_date
-        return if end_date.nil?
-
+      def self.consumption_by_month(aggregate_meter, school)
```

Comment:
There should be specs for this method, that exercises the code for calculating the monthly figures from both amr and manual reads. The system specs aren't really enough to confirm the behaviour